frontend/public/actions/ui.ts

```diff
@@ -167,6 +169,11 @@ export const setActivePerspective = (perspective: string) => {
   return action(ActionType.SetActivePerspective, { perspective });
 };
 
+export const setTopologyFilters = (topologyFilters) => {
```

Comment:
type for `topologyFilters`

Reply:
fixed in https://github.com/openshift/console/pull/3746/commits/2cb083eeccb662abb5c2ff9b4630a026333c3dd1